oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -161,7 +160,7 @@ private BulkProcessor initBulkProcessor() {
     }
 
     protected BiConsumer<BulkRequest, ActionListener<BulkResponse>> requestConsumer() {
-        return (request, bulkListener) -> elasticConnection.getClient().bulkAsync(request, RequestOptions.DEFAULT, bulkListener);
+        return (request, bulkListener) -> elasticConnection.getOldClient().bulkAsync(request, RequestOptions.DEFAULT, bulkListener);
```

Comment:
Can we add a TODO here and anyplace where we are using the oldClient, mentioning that it would have to be removed in future with a link to issue that's preventing us to remove it now ?

Reply:
Done